test/validator.js

```diff
@@ -174,6 +174,14 @@ describe('Validator', () => {
                 id: ['id must be a valid mongo id'],
             });
         });
+
+        const rules2 = { id: 'mongo_id:min=24,max=24' };
```

Comment:
Instead of creating a new `rules2` object and a new `Validator` instance can you add that rule and a new property to the validation object above?

Thanks